cliquet/resource/__init__.py

```diff
@@ -240,6 +248,11 @@ def collection_get(self):
             next_page = self._next_page_url(sorting, limit, lastrecord, offset)
             headers['Next-Page'] = encode_header(next_page)
 
+        records = [
+            {field: val for field, val in record.items() if field in fields}
```

Comment:
Oh !
Do you prefer something like?
```python
records = [
    {field: record.get(field) for field in partial_fields}
    for record in records
]
```

I have no experience with optional fields so that why I hardly see the difference. Is the value of an optional field null or is the field missing? If a field is missing in the record and we ask for this field what should be the behavior?